client/my-sites/site-monitoring/main.tsx

```diff
@@ -167,33 +31,14 @@ export function SiteMetrics() {
 				align="left"
 				className="site-monitoring__formatted-header"
 			></FormattedHeader>
-			<TimeDateChartControls onTimeRangeChange={ handleTimeRangeChange }></TimeDateChartControls>
-			<SiteMonitoringLineChart
-				title={ __( 'Requests per minute & average response time' ) }
-				data={ formattedData as uPlot.AlignedData }
-			></SiteMonitoringLineChart>
-			<div className="site-monitoring__pie-charts">
-				<SiteMonitoringPieChart
-					title="Cache hit/miss"
-					className="site-monitoring-cache-pie-chart"
-					data={ getFormattedDataForPieChart( cacheHitMissFormattedData, {
-						0: 'Cache miss',
-						1: 'Cache hit',
-					} ) }
-				></SiteMonitoringPieChart>
-				<SiteMonitoringPieChart
-					title="PHP vs. static content served"
-					className="site-monitoring-php-static-pie-chart"
-					data={ getFormattedDataForPieChart( phpVsStaticFormattedData, {
-						php: 'PHP',
-						static: 'Static',
-					} ) }
-				></SiteMonitoringPieChart>
-			</div>
-			<SiteMonitoringBarChart
-				title={ __( 'Requests by HTTP Response Code' ) }
-				{ ...statusCodeRequestsProps }
-			/>
+			<SiteMonitoringTabPanel selectedTab={ page } onSelected={ handleTabSelected }>
+				{ () => (
+					<>
+						{ page === 'metrics' && <MetricsTab /> }
+						{ page !== 'metrics' && <LogsTab logType={ page } /> }
```

Comment:
I'm getting a type warning here but I don't know what it is 😕

Reply:
Fixed! https://github.com/Automattic/wp-calypso/pull/80316/commits/76451bae582c5159e2ba6990c9d1c22dd66ac243

There was a bug where `LogsTab` was not restructuring the props object, and it was showing always the Webserver logs. Now it's fixed.

Reply:
Thanks!